common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -103,6 +110,82 @@ def test_teams_enabled_global_staff(self):
         And the correct content should be on the page
         """
         self.set_team_configuration(
-            {u"max_team_size": 10, u"topics": [self.test_topic]}, enroll_in_course=False, global_staff=True
+            {u"max_team_size": 10, u"topics": self.create_topics(1)},
+            enroll_in_course=False,
+            global_staff=True
         )
         self.verify_teams_present(True)
+
+    def test_list_topics(self):
```

Comment:
Similar to one of my comments above, I feel like these tests should be implemented separate from other teams tab actions, since this class could grow enormous to accommodate all the possible things the user could do on the teams tab. For example, we could have a class called `TeamsTopicListTestCase`.